modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/higher_level_reviews_controller.rb

```diff
@@ -22,22 +23,27 @@ class AppealsApi::V1::DecisionReviews::HigherLevelReviewsController < AppealsApi
   )['definitions']['hlrCreateParameters']['properties'].keys
 
   def create
+    deprecate(response: response, link: AppealsApi::HeaderModification::RELEASE_NOTES_LINK, sunset: sunset_date)
```

Comment:
Per our discussion, we should only be adding deprecation headers when the `path_enabled_flag` setting is true.

Reply:
Absolutely, have to push up what I have